web-gateway/app/views/searchItem.scala.html

```diff
@@ -12,6 +12,8 @@
 
     <h2>@message("searchItems")</h2>
 
+	<p>@message("searchInstructions")</p>
```

Comment:
Thanks, will fix this in the next commit. 
I'm not using any IDE at the moment, but the Atom Editor. Is it advisable to use an IDE?